### ISSUE TITLE
You can rewrite shared history

### ISSUE THREAD

**Issue by adymitruk** [emmajane/gitforteams#10]
Your workflow is going to leave some users scratching their heads. You may want to point the user to what needs to be done if you are going to reset a shared branch. My workflow, for example, _relies_ on resetting shared branches. It is a common way to reject Release Candidates.

**Comment by emmajane** [comment]
Just to double check, is this re. the diagrams? I can't tell from this issue which file it's relating to...

**Comment by adymitruk** [comment]
Yes, where you ask "Is this a shared branch". You leave the user no choice but to go and revert. This makes for a mess when you are merging finished features when constructing RCs. There you want to tag each RC and reset the branch, merge some other combination of finished features, force-push. As long as the team is working in the same workflow, they will see their fetch got a non-fast-forward update. They will reset their local branch to that.

There are other times that resetting a shared branch is valid. This is a major one that I described above. For the workflow in your chart, you could add "as long as your team knows, force push".
